.azure-pipelines/steps/run-tests-windows.yml

```diff
@@ -43,7 +43,7 @@ steps:
       # https://bugs.python.org/issue18199
       $env:TEMP = "R:\Temp"
 
-      tox -e py -- -m integration -n auto --duration=5 --junit-xml=junit/integration-test.xml
+      tox -e py -- -m integration -n auto --durations=5 --junit-xml=junit/integration-test.xml
```

Comment:
Soory, it's not clear for me, why must change `--duration` for `--durations` ?

Reply:
From the commit message:

> Note that newer pytest no longer accept --duration as an alias for --durations.

Reply:
perfect @pradyunsg